src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -3351,9 +3356,9 @@ private void SerializeMethodSpecificationSignature(BlobBuilder builder, IGeneric
             }
         }
 
-        private void SerializeCustomAttributeSignature(ICustomAttribute customAttribute, BlobBuilder builder)
+        private void SerializeCustomAttributeSignature(ICustomAttribute customAttribute, IMethodReference constructor, BlobBuilder builder)
```

Comment:
>private void SerializeCustomAttributeSignature(ICustomAttribute customAttribute, IMethodReference constructor, BlobBuilder builder) [](start = 8, length = 131)

It looks like changes to this function are not necessary, consider reverting

Reply:
We can simply call ```Constructor``` method without requesting diagnostics

---
In reply to: [188719435](https://github.com/dotnet/roslyn/pull/26815#discussion_r188719435) [](ancestors = 188719435)